features/support/hooks.rb

```diff
@@ -0,0 +1,5 @@
+at_exit do
```

Comment:
I would prefer after each scenario.... Allowing tests to be agnostic of each other.